### ISSUE TITLE
Add NPP "path to target" feature

### ISSUE THREAD

**Issue by angband-import** [angband/angband#565]
**Reported by takkaria on 21 Mar 2007 14:27 UTC**
This displays a line when targetting which indicates where the projectile/spell will actually end up.

**Comment by angband-import** [comment]
**Modified by takkaria on 3 Jun 2007 16:03 UTC**

**Comment by angband-import** [comment]
**Modified by magnate on 26 Jul 2009 12:38 UTC**

**Comment by angband-import** [comment]
**Modified by magnate on 5 Nov 2009 23:10 UTC**

**Comment by angband-import** [comment]
**Comment by MarbleDice on 28 Nov 2009 16:52 UTC**
Might also be handy to truncate and re-color the path line if the current targeted tile is out of range of the bow, wand, spell being used.

**Comment by angband-import** [comment]
**Comment by magnate on 23 Apr 2011 23:36 UTC**
Done in staging in [r7d1458c], but would be nice to have the feature MarbleDice suggested.

**Comment by angband-import** [comment]
**Comment by magnate on 24 Apr 2011 16:35 UTC**
Ok, that feature cannot be added until #1328 is done and target.c knows about range.

**Comment by angband-import** [comment]
**Comment by myshkin on 5 May 2011 05:15 UTC**
Fixed in master [712c297], but without !MarbleDice's suggestion. Shall we close this, or leave it open?

**Comment by angband-import** [comment]
**Comment by magnate on 6 May 2011 08:22 UTC**
I think we can close it, as the original backport has been completed. MD's suggestion is noted on #1328 and should be a trivial change once that ticket is done.
